source/linux/permissions/reading.rst

```diff
@@ -0,0 +1,37 @@
+Reading Permissions
```

Comment:
This could cause some misunderstandings, something like `How to read permissions` could be better.